docker-compose.yml

```diff
@@ -1,27 +1,27 @@
 services:
-  frontend:
+  web:
     restart: always
     build:
-      context: ./
-      dockerfile: ./frontend/Dockerfile
+      context: ./apps/
+      dockerfile: ./web/Dockerfile
     ports:
       - 80:3000
     volumes:
-      - ./frontend/.env:/app/frontend/.env
+      - ./web/.env:/app/web/.env
     depends_on:
-      - backend
+      - api
 
-  backend:
+  api:
     restart: always
     build:
-      context: ./
-      dockerfile: ./backend/Dockerfile
+      context: ./apps/
```

Comment:
zły kontekst, ale to w osobnej mr'ce chyba poprawimy

Reply:
Docker będzie cały od nowa po zrobieniu turbo, tak jak pisałem